server/src/main/java/org/opensearch/index/translog/transfer/BaseTranslogTransferManager.java

```diff
@@ -99,46 +101,78 @@ public ShardId getShardId() {
         return this.shardId;
     }
 
+    /**
+     * Transfers the translog checkpoint snapshots to a remote store.
+     *
+     * <p>This method starts the transfer of the translog checkpoint snapshots, which includes the translog
+     * and checkpoint files, to a remote store location. The snapshot is represented by a list of generational
+     * snapshots, and the destination paths are specified in a map of blob paths.
+     *
+     * @param generationalSnapshotList A list of generational snapshots representing the translog checkpoint snapshot.
+     * @param blobPathMap              A map containing the blob paths for the remote storage destinations.
+     * @param latchedActionListener    A latched action listener for listening the transfer progress.
+     * @param writePriority            The write priority for the transfer operation.
+     */
+    abstract void transferTranslogCheckpointSnapshot(
+        Set<TranslogCheckpointSnapshot> generationalSnapshotList,
+        Map<Long, BlobPath> blobPathMap,
+        LatchedActionListener<TranslogCheckpointSnapshot> latchedActionListener,
+        WritePriority writePriority
```

Comment:
Do we really need this to be part of the interface. There is only 1 value which is HIGH. Lets remove it and have HIGH being passed from both the implementations.

Reply:
Done.